src/framework/chart/LineChart.jsx

```diff
@@ -82,10 +119,20 @@ export default class LineChart extends Component {
 
     const duration = updateImmediately ? 0 : transitionDuration;
     const marginBottom = 14;
-    const marginRight = yAxisLabelWidth;
+    const marginTop = 5;
     const width = this.$lineChart.width();
-    const availableWidth = width - marginRight;
-    const availableHeight = height - marginBottom;
+    const yAxisMax = height - marginBottom;
+    const yAxisTickSize = width - yAxisLabelWidth;
+
+    let marginLeft = 5;
+    let marginRight = yAxisLabelWidth;
+
+    if (props.isYAxisLeft) {
+      marginLeft = yAxisLabelWidth;
+      marginRight = 5;
+    }
```

Comment:
I added `xAxisLabelHeight` since this can be influenced from the outside (like yAxisLabelWidth) but the margins just make sure that there is no overflow that is hidden. The space around the chart can and should be controlled by the surrounding component.